sw/device/lib/drivers/rv_timer/rv_timer.c

```diff
@@ -0,0 +1,372 @@
+// Copyright EPFL contributors.
```

Comment:
this file seems to be taken from lowRisc, thus you should not change the licence